### ISSUE TITLE
Don't output to stdout, at least in watch-exec mode

### ISSUE THREAD

**Issue by risicle** [cachix/cachix#362]
Cachix (0.6.0) appears to print `Watching /nix/store for new store paths ...` and associated messages to its stdout when running in `watch-exec` mode. This ruins the ability to use the stdout of the wrapped program, which is an issue if you're doing something as simple as

```
cachix watch-exec mycache -- nix-build ... > outputs.out
```

Ideally this status information would go to stderr, but even adding a `--quiet` mode would help here.

Edit: for now, I'm getting away with:

```
  cachix_stderr = pkgs.cachix.overrideAttrs (oldAttrs: {
    patchPhase = oldAttrs.patchPhase + ''
      find ./ -type f -exec sed -i -e 's|putText|putErrText|g' {} \;
    '';
  });
```

**Comment by domenkozar** [comment]
Agreed

**Comment by risicle** [comment]
Updated hack:

```
  cachix_stderr = pkgs.cachix.overrideAttrs (oldAttrs: {
    patchPhase = oldAttrs.patchPhase + ''
      find ./ -type f -exec sed -i \
        -e 's|\bputText\b|putErrText|g' \
        -e 's|\bputStr\b|hPutStr stderr|g' \
        {} \;
    '';
  });
```

Needed a little more patching than I thought.

**Comment by domenkozar** [comment]
Will release 0.6.1 with the fix today.

**Comment by domenkozar** [comment]
I've released Cachix 0.6.1 that fixes this!
